doc/source/ray-observability/user-guides/debug-apps/debug-hangs.rst

```diff
@@ -19,16 +18,16 @@ trace is shown. To show native code frames, set the URL parameter ``native=1`` (
    :width: 60%
 
 .. note::
-   If you run Ray in a Docker container, you may run into permission errors when viewing the stack traces. Follow the `py-spy documentation`_  to resolve it.
+   If you run Ray in a Docker container, you may run into permission errors when viewing the stack traces. Follow the `py-spy documentation`_ to resolve it. If you are a KubeRay user, here is the guide for how to :ref:`configure KubeRay and apply the solution <kuberay-pyspy-integration>`.
    
 .. _`py-spy documentation`: https://github.com/benfred/py-spy#how-do-i-run-py-spy-in-docker
 
 
 Use ``ray stack`` CLI command
 ------------------------------
 
-You can run ``ray stack`` to dump the stack traces of all Ray Worker processes on
-the current node. This requires ``py-spy`` to be installed.
+Once ``py-spy`` is installed, you can run ``ray stack`` to dump the stack traces of all Ray Worker processes on
```

Comment:
Should we mention it is already a part of `ray[default]`?

Reply:
Added a note